src/index.ts

```diff
@@ -292,10 +292,10 @@ const TypedDataUtils = {
   /**
    * Signs a typed message as per EIP-712 and returns its keccak hash
```

Comment:
Some of these other inline docs could use improvements too, but I can do that in a separate PR